add osm-sgid trailheads blogpost (#2247)
agrc/gis.utah.gov: _posts/2023-3-13-collaborative-trailheads.md

---
title: "Trailheads, a Collaborative Approach"
author:
  display_name: Greg Bunce
  email: gbunce@utah.gov
date: 2023-03-13 12:00:00
categories:
  - Featured
tags:
  - data
  - trailheads
  - OSM
  - OpenStreetMap
---

> Certain datasets are best managed collectively. These datasets tend to be dynamic and have more than one steward. 

Reviewer: ```suggestion
At UGRC, we serve up datasets from a vast array of sources. Some, like municipal boundaries or voting precincts, are created in-house. Others are pulled from a single steward, like the land ownership data maintained by SITLA. A large part of our work is aggregating datasets from multiple stewards into coherent, state-wide datasets. Roads, parcels, and address points all fall into this category. 

However, there are some datasets that are so broad in their geographic scope and so dynamic in their content that even an aggregation strategy fails to keep up with their changes and fails to capture all the possible data sources.
```

Forgive my barging in, but I thought this might make for a stronger opening :)

Author: nice, thanks jake!

Author: i'm gonna commit it and i may put some of it in my own words.  good call.

We already have an [established OSM workflow]({% link _posts/2022-03-21-introducing-open-source-places.md %}) for the [Open Source Places SGID layer]({% link data/society/open-source-places/index.html %}). This workflow allows us to source the data directly from OSM. Our goal is to eventually incorporate this collective stewardship model into as many suitable datasets as we can. For now, we have identified [trailheads]({% link data/recreation/trails/index.html %}) as the next suitable SGID layer for this collaborative editing model.

### Collaborate Trailheads
{: .flex .flex--left}

The first step is to incorporate the [existing SGID Trailheads](https://opendata.gis.utah.gov/datasets/utah-trailheads/explore) into OSM. Essentially, we need to conflate –or merge– the two datasets to ensure we’re getting the best of both worlds.

To make this process fun (and easy), we created a simple [map challenge](https://maproulette.org/browse/challenges/28673) and opened it up to the mapping community at large. So, if you’ve ever ventured out in Utah, it’s likely you’ll have something to contribute.

In the map challenge, there are essentially two scenarios:

> - In the first scenario, the SGID trailhead is marked as new to OSM. In that case, you would review the information supplied and either add the missing trailhead to OSM or decline it with a valid reason.
> - In the second scenario, the SGID trailhead is marked as not new to OSM. In this case, you're working on a task to review (and possibly update) existing OSM data. The task is to look at both features and ensure that the most current information is in OSM.

Reviewer 2: ```suggestion
- In the first scenario, the trailhead is marked as new to OSM. In that case, you would review the information supplied and either add the missing trailhead to OSM or decline it with a valid reason.
- In the second scenario, the trailhead is marked as not new to OSM. In this case, you're working on a task to review (and possibly update) existing OSM data. The task is to review the feature and ensure that the most current information is in OSM.
```
I removed the `SGID` reference here because it's not necessary an SGID feature that you're looking at.  Also, "both features" doesn't really make sense because the MapRoulette task is really just reviewing the single location.  It's either and existing OSM feature or a feature that needs to be added (just that Map Roulette task has some info from a UGRC feature).

Author: good call.  thx

Three short vidoes are linked below that walk you through the map challenge.

### What’s Next for the Trailheads Dataset?
{: .flex .flex--left}

Once the map challenge is complete, we’ll then direct all maintenance for the trailheads layer to OSM. 

At that point, we will source the SGID Trailheads layer from OSM much like we do with the SGID Open Source Places layer. This involves bringing the OSM data in-house, running a few validations and quality checks, and then serving it through the SGID so it can be consumed as a GIS layer. 

> The beauty of this collective stewardship model is that, collectively, we have the tools to keep the data current.

Reviewer 2: ```suggestion
The beauty of this stewardship model is that, collectively, we have the tools to keep the data current.
```
suggestion: eliminate some redundancy

Author: i like it!

As we move forward with this model, we will also be working with the authoritative stewards to see how this project fits into similar initiatives that are already underway such as the [OSM for Government](https://wiki.openstreetmap.org/wiki/OpenStreetMap_for_Government) and the [Public Domain Map](https://publicdomainmap.org/).

At UGRC, our ultimate goal is to serve relevant, high-quality data that can be used across platforms. By moving certain layers to OSM, we unlock their potential and open them up to the larger mapping community. With increased visibility they become better and more accessible to a variety of users and decision makers.

We hope you’ll join us in this effort.

### Get Involved in the Map Challenge!
{: .flex .flex--left}

If you’d like to contribute to the project, you can start by watching our three short how-to videos. 

> - How to [create an OSM account and link it to the map challenge](https://vimeo.com/807581370)

Author: done

> - How to [add a new trailhead to OSM](https://vimeo.com/807595086)

Reviewer 3: **todo**: you mention a link to the blog post in the show notes that is missing. 

a link to the previous and next videos would be great.

Author: good idea

Author: done

> - How to [update an existing OSM trailhead](https://vimeo.com/807604623)

Reviewer 3: **thought**: do you want me to put these videos in the agrc vimeo account?

Reviewer 3: we should add the urls to open street map and map roulette in the "show notes"

Author: sure!

Author: let me know if i need to do anything for that.

Reviewer 3: You would need the source videos to upload to `utahagrc` unless you can transfer them. If you don't have access then I would need to be able to get the source videos to upload them.

Author: done

### Follow Along with the Project
{: .flex .flex--left}

> You can follow along with this project by visiting our [collaborative trailheads GitHub repository](https://github.com/agrc/collaborative-trailheads) or by reaching out to {% capture contact %}{% include contact.html subject=page.title contact=site.data.contacts.agrc text=' ' hide-punctuation=true %}{% endcapture %}

Reviewer 2: Does including the phone number make sense in this case?  I think it just sends people to the DTS Help Desk, where they'd help them create a SN ticket.

Reviewer 3: it's kind of the standard for everything. i'd try to overlook it.

Reviewer 2: Yeah...I was tempted to just overlook it.

Author: i agree.  it's silly.  i'll see what i can do.

Author: i'm not even convinced phones make phone calls these days.

Reviewer 3: the only way to do it is to remove the phone number from the contact

Author: we probably don't want to do that.

Author: i can live with it.

### OpenStreetMap and Collective Stewardship
{: .flex .flex--left}

At UGRC, we see [OpenStreetMap](https://www.openstreetmap.org/) (OSM) as an ideal platform for managing and maintaining datasets such as trails, trailheads, campgrounds, and a variety of other layers - because who better to steward this data than all of us? 

OpenStreetMap (OSM) is a collaborative platform for collectively editing data in real-time. In the case of trailheads, mapping enthusiasts can work side-by-side (so to speak) along the data stewards to keep the data up-to-date.  

We already have an [established OSM workflow]({% link _posts/2022-03-21-introducing-open-source-places.md %}) for the [Open Source Places SGID layer]({% link data/society/open-source-places/index.html %}). This workflow allows us to source the data directly from OSM. Our goal is to eventually incorporate this collective stewardship model into as many suitable datasets as we can. For now, we have identified [trailheads]({% link data/recreation/trails/index.html %}) as the next suitable SGID layer for this collaborative editing model.

> ### Collaborate Trailheads

Reviewer: Should this be "Collaborative"? It sounds like an imperative right now (Collaborate, Trailheads!  ... Hmm, is a trailhead a person who's always on the trails? Like a deadhead?)

```suggestion
### Collaborative Trailheads
```

Reviewer: Otherwise, I like it! I'm a fan of having an image or two to break up the text, but that takes some time to implement and I know Matt is anxious to get the the newsletter out.
